lib/load.php

```diff
@@ -15,6 +15,7 @@
 	require dirname( __FILE__ ) . '/class-wp-rest-blocks-controller.php';
 	require dirname( __FILE__ ) . '/class-wp-rest-autosaves-controller.php';
 	require dirname( __FILE__ ) . '/class-wp-rest-block-renderer-controller.php';
+	require dirname( __FILE__ ) . '/class-wp-rest-themes-controller.php';
```

Comment:
We need to wrap this with a `class_exists()` but it can happen in a follow-up PR for all of the controllers: #10526